src/index.js

```diff
@@ -0,0 +1,22 @@
+import _ from 'lodash';
+import './style.css';
+import './font-awesome.css';
+import './main.js';
+import Icon from './app_screenshot.png';
+
+function component() {
+  const element = document.createElement('div');
+
+  // Lodash, now imported by this script
```

Comment:
- [ ] OPTIONAL
To enhance a cleaner syntax kindly endeavor to remove **all** comments from your code to be hosted on a repository.